apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -403,36 +408,51 @@ function QuestionTestsForm({
 }
 
 function QuestionSharing({
```

Comment:
Note that the logic here changed beyond the addition of a "Public source" badge. Originally, we hid the sharing sets if a question was publicly shared. Now that this is configured in the JSON, I figured it made sense to show everything instead of trying to cleverly hide sharing sets when we deemed them irrelevant. This should reduce confusion and set us up for a GUI editor of this.

Reply:
Sounds good.